README.md

```diff
@@ -1,6 +1,6 @@
 # SCSS
 
-[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](<Redirect-URL>)
+[![Lifecycle:Maturing](https://img.shields.io/badge/Lifecycle-Maturing-007EC6)](https://github.com/bcgov/jag-scss)
```

Comment:
is there a reason not using "stable"?
https://lifecycle.r-lib.org/articles/stages.html

Reply:
not at all; therefore, i changed it to stable.